packages/dds/tree/src/feature-libraries/sequence-field/types.ts

```diff
@@ -147,13 +147,11 @@ export type Detach = Remove | MoveOut;
  * Only ever targets empty cells.
  *
  * As a matter of normalization, we only use an AttachAndDetach to represent the following compositions:
- * - Insert ○ Remove
- * - Insert ○ MoveOut
  * - MoveIn ○ Remove
```

Comment:
As a matter of normalization, we only use an AttachAndDetach to represent MoveIn ○ Remove.